cypress/support/pages/order.pageObject.js

```diff
@@ -0,0 +1,57 @@
+import PageObject from '../PageObject';
+
+class OrderPageObject extends PageObject {
+  fillName(name) {
+    cy.get('#name')
+      .type(name);
+  }
+
+  fillCountry(country) {
+    cy.get('#country')
+      .type(country);
+  }
+
+  fillCity(city) {
+    cy.get('#city')
+      .type(city);
+  }
+
+  fillCard(card) {
+    cy.get('#card')
+      .type(card);
+  }
+
+  fillMonth(month) {
+    cy.get('#month')
+      .type(month);
+  }
+
+  fillYear(year) {
+    cy.get('#year')
+      .type(year);
+  }
+
+  clickPurchase() {
+    cy.contains('Purchase')
+      .click();
+  }
+
+  get successModal() {
```

Comment:
Rename 'successModal' to 'getSuccessModal' to follow camelCase naming convention